src/main/java/at/myrecipes/cookBookApi/dto/UserDTO.java

```diff
@@ -0,0 +1,9 @@
+package at.myrecipes.cookBookApi.dto;
+
+public record UserDTO(
+        String username,
+        String email,
+        String firstname,
+        String lastname,
```

Comment:
I think these are not required anymore since we replaced the first and last name with the username - also the frontend does not have any fields for the first and last name anymore while signing-up